### ISSUE TITLE
[Bug]: Windows Explorer Taskbar freezes up and becomes unresponsive 

### ISSUE THREAD

**Issue by Donkful** [TranslucentTB/TranslucentTB#1037]
### Please check for a duplicate first

- [X] I have checked the issue tracker if my bug has already been fixed or answered in another (possibly closed) issue.

### Steps to reproduce the issue

If I leave my laptop open and let it close automatically, without manually shutting down or hitting sleep, and then I sign back on after waking up my PC I come back to the taskbar/windows explorer visually frozen. Opening applications from the taskbar still works if it doesn't require a new icon but the time is frozen and I can't access the start menu. I am able to fix it by restarting explorer by manually opening it through commands and then restarting it * but it is annoying to continuously do so. 

* Last week I had to win+r and open explorer.exe since it didn't show up in task manager but recently it started showing up in task manager again. Nothing substantial was changed between the weeks.

### Expected results

Taskbar should be working normally, I haven't had this issue before but it started happening roughly 1 to 2 weeks ago.

### Actual results

The taskbar freezes up.

### Windows version

Windows 11 Insider builds (builds 22623 and up). Please mention exact build.

### Other details

Windows version is 22631.3810

**Comment by devilbager** [comment]
same here, this might only be happening on laptops

restarting explorer.exe seems to be a temporary way to revert this

Windows version: Windows 11 Home [22621.3737]

**Comment by RameshGokamalla** [comment]
![image](https://github.com/user-attachments/assets/91d0bcbf-4174-4b9a-bea9-f9a3dfe9801f)
same issue

**Comment by FEKIAP** [comment]
In fact, when you click (left or right) on tasks in the taskbar with the mouse, they will react as usual, but when you right-click on the actual blank area of the taskbar or bring up hidden system icons, they do not work. 
I have to call up Windows Task Manager through the physical keyboard in order to restart Windows Explorer to solve the problem.
Some users have reported this issue in the Microsoft Store, but Microsoft has not responded to it. 
Personally, this situation occurs more frequently when I use Wallpaper and TranslucentTB simultaneously (compared to using TranslucentTB alone).
Windows version: Windows 11 Home [22631.3880]

**Comment by 111inhistory** [comment]
The same on 24H2, 26100.1301

**Comment by MrDocument** [comment]
I have the same issue. I can open and close my pinned apps but the taskbar itself is frozen. I have to restart windows explorer every time this happens...

(laptop, 23H2, 22631.3880)

**Comment by Virtualburn** [comment]
Just found this post as I still have this annoying issue, for over 4 months now.  I can't reset laptop as it's an 6 year old work laptop and to be honest I can't be arsed.  But what I will say after reading these comments and seeing where they are posted, mine started when I used an app that would revert the taskbar back to the old menu style.  The Dev completely denied his app would do this although until this point the laptop had no issue of 5 years of use.

The fact the toolbar / taskbar relies on explorer.exe in some cases you mess with it and it breaks something, and this never goes away.  Until a fix is found CTRL + ALT + DEL and restarting explorer.exe is your life going forward.

Maybe this Dev knows what he's doing and can isolate what causes and continues the issue... there's 100's of people out there looking for an answer.

**Comment by Virtualburn** [comment]
Here's a dump file I just created while the issue existed and Explorer was using 25% of system memory.  And the second memory dump is after it was restarted.  Maybe somebody with a bit more knowledge can find something.

HUNG : https://mega.nz/file/tSA1iBKC#vuMkakMDcVzWSnvqZ3eDt40oCImuNe1qvkNhp02y0tc

AFTER RESTART : https://mega.nz/file/oK5lyCzC#rtr8X-SINU96oOr6uhKAg9d2HE3jmR83eRTRG1LFKvw

**Comment by sylveon** [comment]
I looked at your dumps and don't see anything very unusual. Unfortunately, I can't really help with third party mods as I have no clue how they work.

**Comment by Virtualburn** [comment]
> I looked at your dumps and don't see anything very unusual. Unfortunately, I can't really help with third party mods as I have no clue how they work.

Yeah I've removed the offending app a long time ago but the explorer issue remained.  Was just hoping it might lead to some insight related to the issues here.  It seems explorer doesn't like being messed with.

Thanks for looking though.

**Comment by Moderpach** [comment]
This problom is actually exist on clean Windows11 from 22H2. See [Disappearance of Windows 11 Taskbar Icons after Switching Virtual Desktops ](https://answers.microsoft.com/en-us/windows/forum/windows_11-performance/disappearance-of-windows-11-taskbar-icons-after/7117f899-7cff-4c8a-b2ec-43874f461314?messageId=9e652228-41b3-4be4-9378-1105524d5bac). This is a group of issues including icon disappearing and taskbar freezes. And microsoft haven't solved it for a long time. 
TranslucentTB may just make this issue easier to reproduce.

**Comment by vzade-dev** [comment]
issue is maybe slightly different with mine(?) as everything seems to be working as it is and it's just the taskbar being (usually partially) visually frozen. steps to reproduce is basically the same, so I'm going to assume that this might be related to the issue I'm having.

![Image](https://github.com/user-attachments/assets/93357550-5a1e-4288-b7c8-917a009a37e7)

**Comment by Eraya-Warner** [comment]
I've the same issue and **it won't solve no matter what** and is very troublesome, even i did __sfc /scannow in powershell in administrative mode_ but **nothing was found** and I've tried all other methods on youtube and also by Gemini but nothing fixes it only after restarting it would work but that's too much to do like when you're writing something all the data would be gone if that happens in between. So the techies please help 😭🥲🤞🙏

<img width="1163" height="838" alt="Image" src="https://github.com/user-attachments/assets/90caeddd-69d6-4b76-9f6e-619ada0bef9b" />
